app/presenters/content_item_presenter.rb

```diff
@@ -5,6 +5,12 @@ def initialize(content_item)
     @content_item = content_item
   end
 
+  def contributor_links
+    content_item.contributors.map do |content_item|
+      { "title" => content_item.title, "base_path" => content_item.base_path }
```

Comment:
If you're moving this to the presenter, rather remodelling a cut-down content item, I'd change the string keys to symbols, and change them and the params for `govuk_styled_link_list` to be `text` and `path` so it matches `govuk_styled_link`.

Reply:
Done!